apps/src/aichat/views/modelCustomization/PromptCustomization.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {useState, ChangeEvent} from 'react';
+
+import {StrongText} from '@cdo/apps/componentLibrary/typography/TypographyElements';
+import styles from '../model-customization-workspace.module.scss';
+
+const PromptCustomization: React.FunctionComponent = () => {
+  const [temperature, setTemperature] = useState('0.05');
+
+  const handleTemperatureChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setTemperature(event.target.value);
+  };
+
+  return (
+    <div className={styles.verticalFlexContainer}>
+      <div>
```

Comment:
nit: extra div?

Reply:
I think we need this container for flex reasons (ie, to keep all inputs at top of panel and submit button at bottom)